fossunited/fossunited/doctype/foss_event_cfp/foss_event_cfp.py

```diff
@@ -43,8 +43,6 @@ def assign_reviewers(self):
         reviewers = frappe.get_single(
             "FOSS Global CFP Review Settings"
         ).members
-        print(reviewers)
```

Comment:
there is another `print` call in the `foss_chapter_event` module that can be removed

Reply:
This was removed. Will be removing that too